tests/PHPUnit/System/expected/test_CustomEvents_Events.getAction_flat__API.getProcessedReport_day.xml

```diff
@@ -582,9 +582,7 @@
 		<result prettyDate="Saturday, January 9, 2010" />
 	</reportMetadata>
 	<reportTotal>
-		<nb_uniq_visitors>61</nb_uniq_visitors>
-		<nb_visits>64</nb_visits>
-		<nb_events>69</nb_events>
+		<nb_events>63</nb_events>
```

Comment:
summing up all `<nb_events>` in the contained report(s) results in `63`, so the result seems to be correct now. Not yet sure why it was incorrect before, though.